examples/math/render_primitives.rs

```diff
@@ -543,7 +543,7 @@ fn spawn_primitive_3d(
         None, // cone
         None, // conical frustum
         Some(TORUS.mesh().build()),
-        Some(TETRAHEDRON.mesh()),
+        Some(TETRAHEDRON.mesh().into()),
```

Comment:
Couldn't this also be `.build()`? Also, sorry for creating these merge conflicts. 😅

Reply:
Ooh yess, don't worry, it's part of the process 🥰